ui/src/components/Banner.scss

```diff
@@ -0,0 +1,4 @@
+.Banner {
+  width: 100%;
```

Comment:
I haven't checked the "visual" result, but do we really want to take 100% of the header width? Would that be the whole page width?

Reply:
I don't mean the PR to decide on design, it's just about integrating the component.